public/scripts/editor/js/fc/bramble-ui-bridge.js

```diff
@@ -207,7 +207,7 @@ define(function(require) {
       $("#publish-button-cancel").click(hidePublishDialog);
 
       publisher = new Publisher(options);
-      publisher.init(options.project);
+      publisher.init(options.projectDetails);
```

Comment:
I don't think we need to pass in projectDetails anymore, we should just be able to require("project") and use it's values